src/services/product.service.ts

```
@@ -0,0 +1,33 @@
+        const { name, description, price, image_url } = params;
+
+        // Check if product already exists
+        const productExists = await prisma.product.findFirst({
+            where: {
+                name,
+            },
+        });
+
+        if (productExists) throw new HttpException('Product already exists with this name', 400);
+
+        // Create new product
```

ESLint

[error] Replace `········` with `····`

(prettier/prettier)